database/triggers.py

```diff
@@ -34,6 +34,20 @@
 ]
 
 
+tables_with_polygon_geometry = [
+    "plan",
+    "land_use_area",
+    "other_area",
+]
+# tables_with_polygon_geometry = [
+#     klass.__tablename__
+#     for _, klass in inspect.getmembers(models, inspect.isclass)
+#     if inspect.getmodule(klass) == models
+#     and getattr(klass, "geom")
```

Comment:
```
from typing import get_type_hints

and get_type_hints(klass)["geom"] is Mapped[MultiPolygon]
```
might work here. Haven't tried it though.

Reply:
Oh right, forgot to get back to this...